collections-csharp/SortedSetInCSharp/SortedSetInCSharpTests/SortedSetInCSharpUnitTests.cs

```diff
@@ -0,0 +1,184 @@
+using SortedSetInCSharp;
+
+namespace SortedSetInCSharpTests
+{
+    [TestClass]
+    public class SortedSetInCSharpUnitTests
+    {
+        SortedSetMethods sortedSet = new SortedSetMethods();
+        private readonly SortedSet<string> _languages;
+
+        public SortedSetInCSharpUnitTests()
+        {
+            _languages = sortedSet.ProgrammingLanguages();
+        }
+
+        [TestMethod]
+        public void GivenASortedSet_WhenNotEmpty_VerifyCountAndContains()
+        {
+            Assert.IsInstanceOfType(_languages, typeof(SortedSet<string>));
+            Assert.AreEqual(_languages.Count(), 9);
+            Assert.IsTrue(_languages.Contains("Java"));
+            Assert.AreEqual(_languages.First(), "C");
+        }
+
+        [TestMethod]
+        public void GivenASortedSet_WhenNotEmpty_VerifyHasNoDuplicates()
+        {
+            _languages.Add("C");
+            _languages.Add("C++");
+            _languages.Add("C#");
+
+            Assert.IsInstanceOfType(_languages, typeof(SortedSet<string>));
+            Assert.AreEqual(_languages.Count(), 9);
+        }
+
+        [TestMethod]
+        public void GivenASortedSet_WhenNotEmpty_VerifyOverlapsSuccessful()
+        {
+
```

Comment:
Super picky but looks like there is an extra space here. Should be removed.

Reply:
Sorted